### PR TITLE
Changes some 1s and 0s to TRUE and FALSE

### PR DESCRIPTION
**Pull request by ShizCalev**

### CODE REVIEW

**Comment by kevinz000** [comment]
oh boy time to have all of my major prs killed by merge conflicts!!!!

**Comment by Armhulen** [comment]
get good!

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -285,7 +285,7 @@
 	return ((aiControlDisabled==1) && (!hackProof) && (!isAllPowerCut()));
 
 /obj/machinery/door/airlock/hasPower()
-	return ((secondsMainPowerLost == 0 || secondsBackupPowerLost == 0) && !(stat & NOPOWER))
+	return ((secondsMainPowerLost == FALSE || secondsBackupPowerLost == FALSE) && !(stat & NOPOWER))
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: Wrong here just use !seconds...

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -228,13 +228,13 @@
 		if(src.isElectrified())
 			if(!src.justzap)
 				if(src.shock(user, 100))
-					src.justzap = 1
+					src.justzap = TRUE
 					spawn (10)
-						src.justzap = 0
+						src.justzap = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: Remove src

File: code/game/machinery/doors/airlock.dm [hunk 3]
```diff
@@ -304,7 +304,7 @@
 		if(src.secondsBackupPowerLost < 10)
 			src.secondsBackupPowerLost = 10
 	if(!src.spawnPowerRestoreRunning)
-		src.spawnPowerRestoreRunning = 1
+		src.spawnPowerRestoreRunning = TRUE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: remove src

File: code/game/machinery/doors/airlock.dm [hunk 4]
```diff
@@ -615,9 +615,9 @@
 		t1 += text("Main Power Input wire is cut.<br>\n")
 	if(wires.is_cut(WIRE_POWER2))
 		t1 += text("Main Power Output wire is cut.<br>\n")
-	if(src.secondsMainPowerLost == 0)
+	if(src.secondsMainPowerLost == FALSE)
 		t1 += text("<A href='?src=\ref[];aiDisable=2'>Temporarily disrupt main power?</a>.<br>\n", src)
-	if(src.secondsBackupPowerLost == 0)
+	if(src.secondsBackupPowerLost == FALSE)
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: remove src, also wrong just use !seconds...

File: code/game/machinery/doors/airlock.dm [hunk 5]
```diff
@@ -684,39 +684,39 @@
 
 /obj/machinery/door/airlock/proc/hack(mob/user)
 	set waitfor = 0
-	if(src.aiHacking == 0)
-		src.aiHacking = 1
+	if(src.aiHacking == FALSE)
+		src.aiHacking = TRUE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: remove src

File: code/game/machinery/doors/airlock.dm [hunk 6]
```diff
@@ -684,39 +684,39 @@
 
 /obj/machinery/door/airlock/proc/hack(mob/user)
 	set waitfor = 0
-	if(src.aiHacking == 0)
-		src.aiHacking = 1
+	if(src.aiHacking == FALSE)
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: !aiHacking

File: code/game/machinery/doors/airlock.dm [hunk 7]
```diff
@@ -684,39 +684,39 @@
...
+		src.aiHacking = TRUE
 		to_chat(user, "Airlock AI control has been blocked. Beginning fault-detection.")
 		sleep(50)
 		if(src.canAIControl(user))
 			to_chat(user, "Alert cancelled. Airlock control has been restored without our assistance.")
-			src.aiHacking=0
+			src.aiHacking = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 8]
```diff
@@ -684,39 +684,39 @@
...
 			return
 		else if(!src.canAIHack())
 			to_chat(user, "Connection lost! Unable to hack airlock.")
-			src.aiHacking=0
+			src.aiHacking = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 9]
```diff
@@ -684,39 +684,39 @@
...
 			return
 		to_chat(user, "Fault confirmed: airlock control wire disabled or cut.")
 		sleep(20)
 		to_chat(user, "Attempting to hack into airlock. This may take some time.")
 		sleep(200)
 		if(src.canAIControl(user))
 			to_chat(user, "Alert cancelled. Airlock control has been restored without our assistance.")
-			src.aiHacking=0
+			src.aiHacking = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 10]
```diff
@@ -684,39 +684,39 @@
...
 			return
 		else if(!src.canAIHack())
 			to_chat(user, "Connection lost! Unable to hack airlock.")
-			src.aiHacking=0
+			src.aiHacking = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 11]
```diff
@@ -684,39 +684,39 @@
...
 			return
 		to_chat(user, "Upload access confirmed. Loading control program into airlock software.")
 		sleep(170)
 		if(src.canAIControl(user))
 			to_chat(user, "Alert cancelled. Airlock control has been restored without our assistance.")
-			src.aiHacking=0
+			src.aiHacking = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 12]
```diff
@@ -684,39 +684,39 @@
...
 			return
 		else if(!src.canAIHack())
 			to_chat(user, "Connection lost! Unable to hack airlock.")
-			src.aiHacking=0
+			src.aiHacking = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 13]
```diff
@@ -725,7 +725,7 @@
 		to_chat(user, "Receiving control information from airlock.")
 		sleep(10)
 		//bring up airlock dialog
-		src.aiHacking = 0
+		src.aiHacking = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 14]
```diff
@@ -794,17 +794,17 @@
 					else if(src.aiDisabledIdScanner)
 						to_chat(usr, "You've already disabled the IdScan feature.")
 					else
-						src.aiDisabledIdScanner = 1
+						src.aiDisabledIdScanner = TRUE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 15]
```diff
@@ -794,17 +794,17 @@
...
 				if(2)
 					//disrupt main power
-					if(src.secondsMainPowerLost == 0)
+					if(src.secondsMainPowerLost == FALSE)
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src also just !

File: code/game/machinery/doors/airlock.dm [hunk 16]
```diff
@@ -794,17 +794,17 @@
...
 						src.loseMainPower()
 						update_icon()
 					else
 						to_chat(usr, "Main power is already offline.")
 				if(3)
 					//disrupt backup power
-					if(src.secondsBackupPowerLost == 0)
+					if(src.secondsBackupPowerLost == FALSE)
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src also just !

File: code/game/machinery/doors/airlock.dm [hunk 17]
```diff
@@ -877,7 +877,7 @@
 					if(wires.is_cut(WIRE_IDSCAN))
 						to_chat(usr, "You can't enable IdScan - The IdScan wire has been cut.")
 					else if(src.aiDisabledIdScanner)
-						src.aiDisabledIdScanner = 0
+						src.aiDisabledIdScanner = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 18]
```diff
@@ -1232,7 +1232,7 @@
 	else if(locked)
 		to_chat(user, "<span class='warning'>The airlock's bolts prevent it from being forced!</span>")
 	else if( !welded && !operating)
-		if(beingcrowbarred == 0) //being fireaxe'd
+		if(beingcrowbarred == FALSE) //being fireaxe'd
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: just !

File: code/game/machinery/doors/airlock.dm [hunk 19]
```diff
@@ -1321,23 +1321,23 @@
...
 	if(!density)
-		return 1
-	operating = 1
+		return TRUE
+	operating = TRUE
 	update_icon(AIRLOCK_OPENING, 1)
 	src.set_opacity(0)
 	sleep(5)
-	src.density = 0
+	src.density = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 20]
```diff
@@ -1379,20 +1379,20 @@
...
+	return TRUE
 
 /obj/machinery/door/airlock/proc/prison_open()
 	if(emagged)
 		return
-	src.locked = 0
+	src.locked = FALSE
 	src.open()
-	src.locked = 1
+	src.locked = TRUE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/airlock.dm [hunk 21]
```diff
@@ -1379,20 +1379,20 @@
...
 	if(safe)
 		CheckForMobs()
-	return 1
+	return TRUE
 
 /obj/machinery/door/airlock/proc/prison_open()
 	if(emagged)
 		return
-	src.locked = 0
+	src.locked = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: src

File: code/game/machinery/doors/windowdoor.dm [hunk 22]
```diff
@@ -136,19 +136,19 @@
 		if(emagged)
 			return 0
 	if(!src.operating) //in case of emag
-		src.operating = 1
+		src.operating = TRUE
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: src

File: code/game/machinery/doors/windowdoor.dm [hunk 23]
```diff
@@ -136,19 +136,19 @@
...
 	if(!src.operating) //in case of emag
-		src.operating = 1
+		src.operating = TRUE
 	do_animate("opening")
 	playsound(src.loc, 'sound/machines/windowdoor.ogg', 100, 1)
 	src.icon_state ="[src.base_state]open"
 	sleep(10)
 
-	src.density = 0
+	src.density = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: src

File: code/game/machinery/doors/windowdoor.dm [hunk 24]
```diff
@@ -136,19 +136,19 @@
...
 //	src.sd_set_opacity(0)	//TODO: why is this here? Opaque windoors? ~Carn
 	air_update_turf(1)
 	update_freelook_sight()
 
 	if(operating == 1) //emag again
-		src.operating = 0
+		src.operating = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: src

File: code/game/machinery/doors/windowdoor.dm [hunk 25]
```diff
@@ -160,17 +160,17 @@
 	if(forced < 2)
 		if(emagged)
 			return 0
-	src.operating = 1
+	src.operating = TRUE
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: src

File: code/game/machinery/doors/windowdoor.dm [hunk 26]
```diff
@@ -160,17 +160,17 @@
...
 		if(emagged)
 			return 0
-	src.operating = 1
+	src.operating = TRUE
 	do_animate("closing")
 	playsound(src.loc, 'sound/machines/windowdoor.ogg', 100, 1)
 	src.icon_state = src.base_state
 
-	src.density = 1
+	src.density = TRUE
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: src

File: code/game/machinery/doors/windowdoor.dm [hunk 27]
```diff
@@ -160,17 +160,17 @@
...
 	air_update_turf(1)
 	update_freelook_sight()
 	sleep(10)
 
-	src.operating = 0
+	src.operating = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: src

File: code/game/machinery/robot_fabricator.dm [hunk 28]
```diff
@@ -127,7 +127,7 @@ Please wait until completion...</TT><BR>
 			var/building = text2path(build_type)
 			if (!isnull(building))
 				if (src.metal_amount >= build_cost)
-					src.operating = 1
+					src.operating = TRUE
```
Comment (Cyberboss) [inline code/game/machinery/robot_fabricator.dm]: src

File: code/game/machinery/robot_fabricator.dm [hunk 29]
```diff
@@ -142,7 +142,7 @@ Please wait until completion...</TT><BR>
 							src.being_built.loc = get_turf(src)
 							src.being_built = null
 						src.use_power = IDLE_POWER_USE
-						src.operating = 0
+						src.operating = FALSE
```
Comment (Cyberboss) [inline code/game/machinery/robot_fabricator.dm]: src

File: code/game/objects/items/toys.dm [hunk 30]
```diff
@@ -247,13 +247,13 @@
...
 			if(hacked) // That's right, we'll only check the "original" "sword".
-				newSaber.hacked = 1
+				newSaber.hacked = TRUE
 				newSaber.item_color = "rainbow"
 			qdel(W)
 			qdel(src)
 	else if(istype(W, /obj/item/device/multitool))
-		if(hacked == 0)
-			hacked = 1
+		if(hacked == FALSE)
```
Comment (Cyberboss) [inline code/game/objects/items/toys.dm]: !hacked

File: code/modules/admin/verbs/mapping.dm [hunk 31]
```diff
@@ -122,11 +122,11 @@ GLOBAL_LIST_INIT(admin_verbs_debug_mapping, list(
...
 	set name = "Intercom Range Display"
 
-	var/static/intercom_range_display_status = 0
+	var/static/intercom_range_display_status = FALSE
 	if(intercom_range_display_status)
-		intercom_range_display_status = 0
+		intercom_range_display_status = FALSE
 	else
-		intercom_range_display_status = 1
+		intercom_range_display_status = TRUE
```
Comment (Cyberboss) [inline code/modules/admin/verbs/mapping.dm]: interblahblah = !interblahblah

File: code/modules/atmospherics/gasmixtures/gas_types.dm [hunk 32]
```diff
@@ -74,7 +74,7 @@ GLOBAL_LIST_INIT(hardcoded_gases, list("o2","n2","co2","plasma")) //the main fou
 	specific_heat = 40
 	name = "Nitrous Oxide"
 	gas_overlay = "nitrous_oxide"
-	moles_visible = 1
+	moles_visible = TRUE
```
Comment (Cyberboss) [inline code/modules/atmospherics/gasmixtures/gas_types.dm]: Wrong

File: code/modules/food_and_drinks/kitchen_machinery/gibber.dm [hunk 33]
```diff
@@ -162,7 +162,7 @@
 	use_power(1000)
 	visible_message("<span class='italics'>You hear a loud squelchy grinding sound.</span>")
 	playsound(src.loc, 'sound/machines/juicer.ogg', 50, 1)
-	src.operating = 1
+	src.operating = TRUE
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/kitchen_machinery/gibber.dm]: src

File: code/modules/food_and_drinks/kitchen_machinery/gibber.dm [hunk 34]
```diff
@@ -231,5 +231,5 @@
 					new gibtype(gibturf,i)
 
 		pixel_x = initial(pixel_x) //return to its spot after shaking
-		src.operating = 0
+		src.operating = FALSE
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/kitchen_machinery/gibber.dm]: src

File: code/modules/food_and_drinks/kitchen_machinery/processor.dm [hunk 35]
```diff
@@ -251,7 +251,7 @@
 	if(src.contents.len == 0)
 		to_chat(user, "<span class='warning'>The processor is empty!</span>")
 		return 1
-	src.processing = 1
+	src.processing = TRUE
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/kitchen_machinery/processor.dm]: src

File: code/modules/food_and_drinks/kitchen_machinery/processor.dm [hunk 36]
```diff
@@ -274,7 +274,7 @@
 			continue
 		P.process_food(src.loc, O, src)
 	pixel_x = initial(pixel_x) //return to its spot after shaking
-	src.processing = 0
+	src.processing = FALSE
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/kitchen_machinery/processor.dm]: src

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Good PR

**Comment by Fox-McCloud** [comment]
Be very careful with this, it's easy to think something is boolean, when it's really not.

**Comment by MoreRobustThanYou** [comment]
cyberboss, just enjoy your wettest dream ever

**Comment by MoreRobustThanYou** [comment]
@Fox-McCloud doesn't matter, TRUE is just 1. FALSE is just 0.

**Comment by ExcessiveUseOfCobblestone** [comment]
@MoreRobustThanYou so then what's the problem with leaving them all 0/1s.

The whole point of the defines is to make it clear to the person reading the code because it's a lot easier to click the 0 key than it is to type FALSE or TRUE. With that said, it's important to not make something that isn't True/False appear as such.

**Comment by MoreRobustThanYou** [comment]
@ExcessiveUseOfCobblestone cleanness

**Comment by ExcessiveUseOfCobblestone** [comment]
WRONG

**Comment by MoreRobustThanYou** [comment]
@ExcessiveUseOfCobblestone consistency and butts

**Comment by DoctoreMad** [comment]
Wait, does any of this actually matter?

**Comment by ShizCalev** [comment]
technicalllllllllllllly

no, not at all.

**Comment by optimumtact** [comment]
This is an absolutely terrible idea

**Comment by optimumtact** [comment]
Are you just converting every single 0 and 1 to true and false? or are you actually checking?

**Comment by ShizCalev** [comment]
Actually checking.

File: code/modules/admin/verbs/mapping.dm [hunk 37]
```diff
@@ -124,7 +124,7 @@ GLOBAL_LIST_INIT(admin_verbs_debug_mapping, list(
 
 	var/static/intercom_range_display_status = FALSE
 	if(intercom_range_display_status)
-		intercom_range_display_status = FALSE
+		intercom_range_display_status = !intercom_range_display_status
 	else
 		intercom_range_display_status = TRUE
```
Comment (Cyberboss) [inline code/modules/admin/verbs/mapping.dm]: No i meant these entire 4 lines -> `intercom_range_display_status = !intercom_range_display_status`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
okay well in that case it's a good idea +1

**Comment by Cyberboss** [comment]
In the future, please don't rebase as it means I have to review the whole thing again ;-;

**Pull request merged**